mods/ts/tilesets/snow.yaml

```diff
@@ -55,6 +55,14 @@ Terrain:
 		Type: Veins
 		Color: 226, 103, 0
 		TargetTypes: Ground
+	TerrainType@Tree:
+		Type: Tree
+		TargetTypes: Ground
+		Color: 255,28,32,36
```

Comment:
Propose an alternative. (It is the value from https://github.com/OpenRA/OpenRA/blob/bleed/mods/cnc/tilesets/snow.yaml#L49)